### PR TITLE
Add COSIGN_YES env var

### PR DESCRIPTION
**Pull request by hectorj2f**
## Description of the change

related to https://github.com/sigstore/helm-charts/issues/484

## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

**Review by priyawadhwa — DISMISSED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
             file_name=${chart##*/}
             chart_name=${file_name%-*}
             digest=$(awk -F "[, ]+" '/Digest/{print $NF}' < helm-push-output.log)
-            cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
+            COSIGN_YES=true cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
           done
         env:
           COSIGN_EXPERIMENTAL: 1
```
Comment (cpanato) [inline .github/workflows/release.yml:83]: maybe drop this one and replace to `COSIGN_YES=true`

Comment (hectorj2f) [reply .github/workflows/release.yml:83]: @cpanato I don't understand your comment. Can you explain what you want to replace here, I am already using COSIGN_YES ?

Comment (cpanato) [reply .github/workflows/release.yml:83]: can we remove the `COSIGN_EXPERIMENTAL: 1` and instead put the COSIGN_YES=true inline with the command add in the `env` section, like

```
...
            cosign sign "ghcr.io/${GITHUB_REPOSITORY}/${chart_name}@${digest}"
          done
        env:
          COSIGN_YES: "true"
```

Comment (hectorj2f) [reply .github/workflows/release.yml]: nvm I now understood, you wanted to replace `COSIGN_EXPERIMENTAL: 1` with `COSIGN_YES`.

**Review by cpanato — APPROVED** [review]
thank you

**Pull request merged**